test/frisk.test.js

```diff
@@ -234,6 +240,229 @@ describe('Express Frisk Middleware', () => {
                 });
                 frisk.validateRequest(testSchemas.restaurantSchema,true)(req,res,Spies.nextReject);
             });
+
+            // This fails right now
+
+        });
+    });
+
+    it.skip('validates arrayOfStrings properly', () => {
```

Comment:
Did you still want to skip this one?

Reply:
Ya I didn't fix it yet.  I'll make another PR today that does an overhaul of the typing and add the enums.  I'll fix it in there.